### DISCUSSION TITLE
Plan webserver on different machine

### DISCUSSION THREAD

**Discussion by ulllupollllo** [plan-player-analytics/Plan#2850] [Q&A] [answered]
I was trying to run my webserver on different machine for security reason,
my proxy server is on my main machine and when i try to host the webserver on the different machine i get this:
![image](https://user-images.githubusercontent.com/82157501/215229088-0ffe1915-c658-4584-8656-ef6435687e52.png)
(I have configured plan as a network with some subserver)

**Comment by AuroraLS3** [comment, marked as answer]
Try setting Plugin.Allow_proxy_to_override_settings to false and then Disable_webserver back to false

**Reply by ulllupollllo** [reply]
and it works but i don't have all the info from the other servers... (i have only the info of the new server)

**Reply by AuroraLS3** [reply]
You can try the new navigation button in this dev build https://github.com/plan-player-analytics/Plan/actions/runs/4015211983

Install it on the server with the web server and Set Plugin.Frontend_BETA to true

**Reply by ulllupollllo** [reply]
now is working thanks!
